plugin/src/main/java/org/autorefactor/refactoring/ASTHelper.java

```diff
@@ -195,6 +196,32 @@ public int compare(ASTNode o1, ASTNode o2) {
         }
     }
 
+    private static final class VariableDeclarationIdentifierVisitor extends ASTVisitor {
+        private Set<String> variableNames = new HashSet<String>();
+        private ASTNode startNode;
+        private boolean includeInnerScopes;

```

Comment:
Likewise:

```java
        private final ASTNode startNode;
        private final boolean includeInnerScopes;
```

I like `final` on fields because it is much easier to reason about whether this field value can change at all.